demo/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Renuo Upload Client Example</title>
+
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css"
+        integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css"
+        integrity="sha256-3dkvEK0WLHRJ7/Csr0BZjAWxERc5WH7bdeUya2aXxdU= sha512-+L4yy6FRcDGbXJ9mPG8MT/3UCDzwR9gPeyFNMCtInsol++5m3bk2bXWKdZjvybmohrAsn3Ua5x8gfLnbE1YkOg=="
+        crossorigin="anonymous">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/dropzone/4.3.0/basic.min.css">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/dropzone/4.3.0/dropzone.min.css">
+
+  <!--[if lt IE 9]>
+  <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
+  <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
+  <![endif]-->
+</head>
+<body>
+<div class="container">
+  <h1>Renuo Upload Client Examples</h1>
+
+  <div class="renuo-upload dz-clickable" data-apikey="wDDm1o6DMLtSsUYkxlw3TSMQfKGdtx"
+       data-signingurl="//renuo-upload-signing.dev:3003/generate_policy">
+    <div class="dz-default dz-message">
+            <span>Ziehen Sie eine Bild-Datei oder ein PDF hier herein, oder klicken
+                Sie hier um die Datei hochzuladen.</span>
+    </div>
+  </div>
+</div>
+</body>
+<script src="https://cdn.jsdelivr.net/jquery/2.2.3/jquery.min.js"></script>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"
+        integrity="sha384-0mSbJDEHialfmuBBQP6A4Qrprq5OVfW37PRR3j5ELqxss1yVqOtnepnHVP9aJ7xS"
+        crossorigin="anonymous"></script>
+<script src="https://cdn.jsdelivr.net/dropzone/4.3.0/dropzone.min.js"></script>
+<script src="renuo-upload.js"></script>
+
+<script>
+  $(document).ready(function () {
+    var dropzoneOptions = {
+      acceptedFiles: 'image/*,application/pdf',
+      dictDefaultMessage: 'Ziehen Sie eine Bild-Datei oder ein PDF hier herein, oder klicken Sie hier um die Datei hochzuladen.',
+      dictFallbackMessage: 'Sie verwenden einen veralteten Browser was das verwenden des Uploads verhindert.',
+      dictInvalidFileType: 'Hier können nur Bild-Dateien hochgeladen werden.',
+      dictFileTooBig: 'Die Datei ist zu gross.'
+    };
+    new window.RenuoUpload($('.renuo-upload'), dropzoneOptions, function (a, b, c) {
+      console.log(a);
```

Comment:
like @cyrilkyburz already noticed: we could use some more sophisticated logging here (see readme). agree?